### ISSUE TITLE
500 Internal Server Error Server got itself in trouble

### ISSUE THREAD

**Issue by helmarw** [rumbu13/ha-mediabrowser#12]
### System Health details

Homeassistant Docker on Synology NAS DS1821+
Core 2024.1.2
Frontend 20240104.0

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

after trying to add this integration, thats all im getting: 500 Internal Server Error Server got itself in trouble

no idea where to start. log-file not really helpful. im looking at the files listed but no clue why there si a problem
and idea ?

### Reproduction steps

follow installation instructions
it shows up when searching for MediaBrowser but cannot add Integration

### Debug logs

```text
2024-07-09 11:43:12.118 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration mediabrowser which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
  File "/config/custom_components/mediabrowser/config_flow.py", line 90, in async_step_user
  File "/config/custom_components/mediabrowser/discovery.py", line 22, in discover_mb
  File "/config/custom_components/mediabrowser/discovery.py", line 31, in _discover_message
  File "/config/custom_components/mediabrowser/config_flow.py", line 90, in async_step_user
  File "/config/custom_components/mediabrowser/discovery.py", line 22, in discover_mb
  File "/config/custom_components/mediabrowser/discovery.py", line 31, in _discover_message
```

**Comment by ghost-03** [comment]
I don't think this project is maintained, but I had a similar problem, also with Docker on a Synology NAS. What helped is that I modified the 'discovery.py' file in the plugin, and changed the hostname in line 32 from socket.gethostname() to "localhost". After a restart, the plugin worked as expected. Possibly either the NAS or the Docker confused Python. Hope this helps.

**Comment by Cyberdr3am** [comment]
Hi i use this extention a lot but when i whant to add another server i have got an error.

The configuration feed could not be loaded: 500 Internal Server Error Server got itself in trouble

Can someone have a fix ?

Thx
